tools/sync-npm-deps-to-tsc-projects.ts

```diff
@@ -19,16 +19,19 @@ const main = async (argv: string[], env: NodeJS.ProcessEnv) => {
   }
   const TS_CONFIG = "tsconfig.json";
   const PACKAGE_JSON = "package.json";
+  const LERNA_JSON = "lerna.json";
   const SCRIPT_DIR = __dirname;
   const PROJECT_DIR = path.join(SCRIPT_DIR, "../");
   console.log(`SCRIPT_DIR=${SCRIPT_DIR}`);
   console.log(`PROJECT_DIR=${PROJECT_DIR}`);
-
-  const pkgJsonGlobPatterns = lernaJson.packages.map((it) =>
+  const lernaJson = JSON.parse(
```

Comment:
@aldousalvarez Looking much better now, thank you! Last nit-pick is to pull these two function calls out to their own lines so that it's easier to debug what crashed if all we have is that there was some exception thrown on a particular line.

Reply:
@petermetz Hello, already done with the latest [commit](https://github.com/hyperledger/cactus/pull/2164/commits/997eab68410e27442aaf01ffd4cc0bfb19cf683b). is this the two function calls that you mentioned that should be pulled out to their own? 

const pkgJsonGlobPatterns = lernaJson.packages.map((it: string) =>
  "./".concat(it).concat(`/${PACKAGE_JSON}`),
);

const tsConfigJsonGlobPatterns = lernaJson.packages.map((it: string) =>
  "./".concat(it).concat(`/${TS_CONFIG}`),
);

Reply:
@aldousalvarez No, I meant the function calls on the line that this comment is specifically attached to:
`const lernaJson = JSON.parse(await readFile(PROJECT_DIR + LERNA_JSON, "utf-8"));`

Those other two snippets you quoted look good as-is.